angelspetrochem/prototypes/petrochem-global-override.lua

```diff
@@ -27,7 +27,7 @@ require("prototypes.global-override.bobplates")
 require("prototypes.global-override.bobrevamp")
 require("prototypes.global-override.bobwarfare")
 require("prototypes.global-override.bobassembly")
-require("prototypes.global-override.bobtech")
+--require("prototypes.global-override.bobtech")
```

Comment:
Just remove the whole file itself, rather than leaving it empty?

Reply:
well … I thought for the future we might need it for other stuff, and so we already know where to put it?

Reply:
hmm.. if you leave the file, at least don't comment out the require statement, or we might have a long debug session XD

Reply:
ok